features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -225,6 +215,17 @@
     <bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial.linuxsysfs/${project.version}</bundle>
   </feature>
 
+  <feature name="openhab-core-ephemeris" version="${project.version}">
+    <feature>openhab-core-base</feature>
+    <bundle>mvn:org.threeten/threeten-extra/1.4</bundle>
+    <bundle>mvn:de.jollyday/jollyday/0.5.7</bundle>
```

Comment:
For all external bundle dependencies (I identified we missed it for JNA on migration) we create an target platform feature that is referenced by the core features.